util/linux_install.sh

```diff
@@ -126,6 +140,25 @@ elif grep ID /etc/os-release | grep -qE "opensuse|tumbleweed"; then
 		wget \
 		zip
 
+elif grep ID /etc/os-release | grep -q slackware; then
+	echo $SLACKWARE_WARNING
+	read -rp "Would you like to install them with sudo and sboinstall (y/N)? " answer
```

Comment:
`read -p` is non-POSIX, so replace it with `printf` followed by `read`.
```suggestion
  printf "\nProceed (y/N)? "
	read -r answer
```
The “sudo and sboinstall” text has been moved to the new `SLACKWARE_WARNING` message. This will also make this part of the script the same as what I did for Gentoo in #4395.